File: SPACE TRIANGLE PATTERN/code3.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);
	int temp = 64 + row*2;

	for(int i = 1 ; i <= row ; i++){

		for(int a = 0 ; a < row - i ; a++){

			printf(" \t");
		}

		for(int j = 0; j < i ; j++){

			printf("%c\t",temp--);

		}

		printf("\n\n");

	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code3.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		for(int i = 1; i <= row ; i++){
			int temp = i;

			for(int j = 0; j < row ; j++){

				System.out.print(temp+"		");
				temp += 4;
			}

				System.out.print("\n\n");

		}

	}

}
